### DISCUSSION TITLE
Attributi avanzati via API

### DISCUSSION THREAD

**Discussion by alessiorigato94** [fattureincloud/api#284] [Q&A] [answered]
Buongiorno,
via API, quotidianamente, verifico la lista delle fatture ricevute. 
Tuttavia nella lista non si riesce a capire come recuperare un attributo avanzato. Nella DOC non ho trovato alcuna informazione a riguardo.

La chiamata è abbastanza semplice:
`...../received_documents?type=expense&fieldset=detailed&sort=-date`.

Il cliente nell'attributo avanzato "2.1.15" immette un codice. 
C'è un modo per aggiungere nel JSON di risposta quel dato?

Grazie infinite

**Comment by valmoz** [comment, marked as answer]
Buongiorno @alessiorigato94,
Intendi il campo 2.2.1.15? perché dal [file ufficiale](https://www.fatturapa.gov.it/export/documenti/fatturapa/v1.3/Rappresentazione-tabellare-fattura-ordinaria.pdf) non vedo un 2.1.15.
Se non sbaglio il campo 2.2.1.15 dovrebbe essere all'interno di items_list, nella [lista dei campi dei fieldset](https://developers.fattureincloud.it/docs/basics/additional-capabilities/customize-response/#-available-fieldsets) io vedo che il fieldset "detailed" contiene items_list; dovrebbe essere contenuto nel campo ei_raw dei vari item.
Ti chiederei di controllare se il campo si trova nel punto indicato.

Puoi trovare maggiori informazioni ai seguenti link:
- [E-Invoice XML Customization](https://developers.fattureincloud.it/docs/guides/e-invoice-xml-customisation)
- [Customize response](https://developers.fattureincloud.it/docs/basics/additional-capabilities/customize-response)

A disposizione per ulteriori chiarimenti
Mauro

**Reply by alessiorigato94** [reply]
Mi scuso dell'errore. Il codice corretto è "1.2.6" ( RiferimentoAmministrazione).

Dalla fattura elettronica ricevuta vedo che il dato è stato messo correttamente.
Tuttavia dal JSON di risposta "items_list" è NULL....

**Reply by valmoz** [reply]
Mi scusi, ho notato ora che questo è un received document e non un issued document.

In tal caso non abbiamo a disposizione il campo parsato all'interno del received document. Deve utilizzare il campo attachment_url per scaricare l'XML del documento e poi procedere a parsare l'XML ricevuto in autonomia.

A disposizione
Mauro

**Reply by valmoz** [reply]
Buongiorno @alessiorigato94 
abbiamo risposto alle sue domande?

Grazie
Mauro
